chart/postgres-catalog/requirements.yaml

```diff
@@ -0,0 +1,21 @@
+dependencies:
+- name: postgres-stash
+  version: 9.6
+  repository: "@appscode"
```

Comment:
Replace it with `https://charts.appscode.com/stable/`. We can't assume that everyone is going to call the repo appscode.